app/helpers/mailer_helper.rb

```diff
@@ -30,12 +30,10 @@ def add_reason(text, reason)
   end
 
   def step_status_icon(step)
-    if step.status == "actionable"
-      "emails/numbers/icon-number-" + (step.position - 1).to_s + ".png"
```

Comment:
ok sounds good - generally I am opposed to keeping things in source control that are unused bc that can end up piling up, so let's try and remember to remove these if we aren't using them in a few weeks